deploy/evmos/120_check_EvmosBTCTokens.ts

```diff
@@ -0,0 +1,43 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+import { isTestNetwork } from "../../utils/network"
+import { BigNumber } from "ethers"
+
+const USD_TOKENS_ARGS: { [token: string]: any[] } = {
+  WBTC: ["Wrapped Bitcoin", "WBTC", "8"],
+  RENBTC: ["renBTC", "RENBTC", "8"],
```

Comment:
symbol is `renBTC` as per their contract https://etherscan.io/token/0xeb4c2781e4eba804ce9a9803c67d0893436bb27d#readProxyContract

Reply:
Token symbols here are used for json filenames locally so should be ok